docs/docs/basil.md

```diff
@@ -128,16 +128,32 @@ Let's start a local ethereum network:
 ganache-cli --deterministic
 ```
 
-This will print 10 accounts. Copy the address of the first one, and then back into the initial terminal, export it as the `OWNER` because it will be useful for us later:
+This will print 10 accounts. Copy the address of the first one and then go back to the terminal window that you used to launch the Ganache client.  Export the address you just copied to an environment variable named `OWNER`.  You will need easy access to that address as you complete this tutorial:
```

Comment:
This phrase could indeed be improved, but please note that the "going back" is not to the terminal running the Ganache client, but the other, where the initial commands were run, and is at this point free to run the next steps.

Reply:
Agreed.  Changed text to tell the user to go back to the terminal window that they used to create the Basil project.